demo-shell/src/app/components/cloud/community/community-processes-cloud.component.scss

```diff
@@ -0,0 +1,21 @@
+:host {
+    height: 100%;
+    min-height: 100%;
+    min-width: 100%;
+    width: 100%;
+}
+
+.app-cloud-process-list-container {
+    place-content: stretch space-between;
+    align-items: stretch;
+    margin-top: 2px;
+}
+
+:host, .app-cloud-process-list-container {
```

Comment:
same as above, better getting rid of the ":host" here

Reply:
done